src/epics/sync.js

```diff
@@ -40,7 +40,8 @@ export const getNavEpic = routes => {
  */
 export const resetLocationEpic = (action$, store) =>
 	action$.ofType('CONFIGURE_AUTH')  // auth changes imply privacy changes - reload
-		.filter(({ meta }) => !meta)  // throw out any server-side actions
+		.filter(({ meta }) => !meta)  // throw out any cases where sync is specifically suppressed
+		.filter(() => !store.getState().auth.oauth_token)  // ignore cases where auth tokens are not set
```

Comment:
this is the only new code - the rest is just renaming vars so that their purpose is more explicit